cpp/tests/streams/stream_compaction_test.cpp

```diff
@@ -233,3 +234,232 @@ TEST_F(StableDistinctKeepFirstLastNone, InputWithNaNsUnequal)
     CUDF_TEST_EXPECT_TABLES_EQUAL(expected, *result);
   }
 }
+
+struct StreamCompactionTest : public cudf::test::BaseFixture {};
```

Comment:
Given there's already a fixture for stable distinct, for now I just made a `StreamCompactionTest` fixture, but I understand its probably not the best way. One way can be that we include stable distinct tests within `StreamCompactionTest` fixture. What do everyone think ? Thanks.

Reply:
I would prefer to add the stable distinct tests to the new `StreamCompactionTest`.
Note that the stream tests are more about testing all code paths pass the stream parameter through and less about correctness in the output which should already have been checked by the normal unit tests.

Reply:
That depends. Are they testing different code paths where the stream is passed through?
Hopefully that makes sense. We want to make sure any internal divergent (CPU) pass the stream correctly.
If these stream tests are all processing through the same code path then I think we can pare them down accordingly.